Sources/Host/Linux/PTrace.cpp

```diff
@@ -281,23 +273,12 @@ ErrorCode PTrace::step(ProcessThreadId const &ptid, ProcessInfo const &pinfo,
       return error;
   }
 
-  if (wrapPtrace(PTRACE_SINGLESTEP, pid, nullptr, signal) < 0)
-    return Platform::TranslateError();
-
-  return kSuccess;
+  return super::step(ptid, pinfo, signal);
```

Comment:
Explicitly write `Address()` for `super::step`'s `address` parameter? At first glance, it looks like you forgot to pass the `address` to `super::step`.